CHANGELOG-add-status-icon-to-multiassay-listing.md

```diff
@@ -0,0 +1 @@
+- Added status icon to the processed datasets to differntiate between different runs 
```

Comment:
```suggestion
- Added status icon to the processed datasets to differentiate between different runs.
```

Minor spelling fix/added comma at the end - if you can re-request review after fixing this, I'll sign off on it 😄